scripts/overlap/kubectl-kubestellar-ensure-wmw

```diff
@@ -130,19 +131,16 @@ elif [ "$want_kube" == false ] && kubectl --kubeconfig $space_config "${kubectl_
 fi
 }
 
-bind_iff_wanted kubernetes
-bind_iff_wanted apiregistration.k8s.io
-bind_iff_wanted apps
-bind_iff_wanted autoscaling
-bind_iff_wanted batch
-bind_iff_wanted core.k8s.io
-bind_iff_wanted cluster-core.k8s.io
-bind_iff_wanted discovery.k8s.io
-bind_iff_wanted flowcontrol.apiserver.k8s.io
-bind_iff_wanted networking.k8s.io
-bind_iff_wanted cluster-networking.k8s.io
-bind_iff_wanted node.k8s.io
-bind_iff_wanted policy
-bind_iff_wanted scheduling.k8s.io
-bind_iff_wanted storage.k8s.io
-bind_iff_wanted cluster-storage.k8s.io
+# CI expects these two, why?
+bind_iff_wanted kubernetes # ingresses.networking.k8s.io, services.core, deployments.apps, pods.core
```

Comment:
Why do we still have the bind_iff_wanted  function defined , and called ?

Reply:
What I don’t understand is that our CI expects two apibindings (bind-apps, bind-kubernetes) to appear in mailboxes. I see such lines in `docs/scripts/generated_script.sh`:
```
# TODO: kubestellar-list-syncing-objects has kcp dependencies. Will remove when controllers support spaces.
test $(kubestellar-list-syncing-objects --api-group apis.kcp.io --api-version v1alpha1 --api-kind APIBinding | grep -cw "name: bind-apps") -ge 2
kubestellar-list-syncing-objects --api-group apis.kcp.io --api-version v1alpha1 --api-kind APIBinding | grep -w "name: bind-kubernetes"
```
But I don’t understand why these apibindings should appear in mailboxes.

Reply:
resolved elsewhere.